### ISSUE TITLE
Serve command fails silently

### ISSUE THREAD

**Issue by TristanCacqueray** [containers/ramalama#285]
Hello,

Running the `serve` command with a wrong model name outputs a container ID that dies instantly without a reason.
Here is an example reproducer:

```ShellSession
$ ramalama serve huggingface://ibm-granite/granite-7b-instruct
6aa6dc6cca217b28656990f69a7c602c28c40f64289d8cdb5c55e6ecccdc0828
$ ramalama ps
CONTAINER ID  IMAGE       COMMAND     CREATED     STATUS      PORTS       NAMES
$
```

I tried to add an helpful message by calling the pull command before serve, but it looks like there is another issue in the glob `f"{args.store}/models/*/{model}"` because the Model class doesn't have a str method, resulting in the path to never match because it looks like:

```raw
ramalama/models/*/<ramalama.huggingface.Huggingface object at 0x7fa1c6e659a0>
```

How would you recommend to fix that issue?

**Comment by ericcurtin** [comment]
I kinda wish we didn't detach by default because of reasons like this

**Comment by TristanCacqueray** [comment]
I meant to add a check before calling `ramalama.run_container` in the Process CLI entrypoint, to ensure the model is pulled before starting the service.

**Comment by ericcurtin** [comment]
The problem with just adding a check for the file, is that only helps debug one very specific issue. Like it is already checked if the file exists just in llama-server and we can't see the output

**Comment by TristanCacqueray** [comment]
Right, so not detaching by default sounds great to me, may I propose such a change?
